sdk/edgezones/arm-edgezones/review/arm-edgezones-api.api.md

```diff
@@ -25,6 +26,21 @@ export type EdgeZonesContext = Client & {
     path: Routes;
 };
 
+// @public
+export function get(context: EdgeZonesContext, subscriptionId: string, extendedZoneName: string, options?: ExtendedZonesGetOptionalParams): Promise<ExtendedZone>;
```

Comment:
Do you enable operationGroup option? Could you share the config with me?

Reply:
"@azure-tools/typespec-ts":
    azureSdkForJs: true
    isModularLibrary: true
    generateMetadata: true
    hierarchyClient: false
    experimentalExtensibleEnums: true
    enableOperationGroup: true
    package-name: "@azure/arm-edgezones"
    package-dir: "arm-edgezones"
    emitter-output-dir: "{project-root}/sdk/edgezones/arm-edgezones"
    flavor: "azure"
    packageDetails:
      name: "@azure/arm-edgezones"
      description: "Microsoft Edgezones Service"

Reply:
@qiaozha there is no operationGroup prefix before the method, is this expected?

Reply:
no, it's fixed in the codegen mongocluster test pr. @kazrael2119 could you file a pr for that fix ?

Reply:
https://github.com/Azure/autorest.typescript/pull/2717